$NetBSD: patch-c__check,v 1.2 2026/02/24 23:09:02 thor Exp $

Do not apply broken -Y handling on SunOS.
Note: Should this also include the -L case like f_check?

--- c_check.orig	2024-01-02 21:25:05.000000000 +0000
+++ c_check
@@ -384,9 +384,11 @@ for flag in $flags; do
         esac
     esac
 
+    if [ $hostos != "SunOS" ]; then
     case "$flag" in -Y*)
         linker_L="$linker_L -Wl,$flag" ;;
     esac
+    fi
 
     case "$flag" in --exclude-libs*)
         linker_L="$linker_L -Wl,$flag"
